homeassistant/components/media_player/kodi.py

```diff
@@ -481,7 +486,14 @@ def name(self):
     @property
     def should_poll(self):
         """Return True if entity has to be polled for state."""
-        return not (self._enable_websocket and self._ws_server.connected)
+        return not (
+            self._enable_websocket
+            and self._ws_server.connected
+            and not (
+                self._item.get('type') == 'channel'
+                and ttime() - self._last_update >= TV_POLL_INTERVAL
```

Comment:
Please don't try and implement a separate poll interval. Just return true here if `self._item.get('type') == 'channel'`. `should_poll` isn't guaranteed to be called again if it returns false at the last update. Also please add a comment explaining why we're polling for TV.

Reply:
It is guaranteed. I checked the code and also asked in dev channel. Please have a look at the code.

I set it to 60 secs because we don't need 10 secs for just title updates.

Reply:
In any case, the extra complication of a secondary poll interval is unnecessary.

Reply:
I can of course remove it but that'd mean 6x more queries just to update the title. Not sure if that's worth it as the logic is already in place and working fine so why not leave it in?